src/Native/Utils.js

```diff
@@ -325,12 +325,17 @@ Elm.Native.Utils.make = function(localRuntime) {
 		throw new Error('Runtime error in module ' + moduleName + ' (' + span + ')' + msg);
 	}
 
+	function formatValue(value) {
```

Comment:
I would bring the bracket down to the next line to match the style of the file, or even just remove the `formatValue` function and inline the check. Otherwise :+1: LGTM!